app/properties/page.tsx

```diff
@@ -22,6 +22,9 @@ export default async function PropertiesPage(){
 
     return(
         <div>
+            <div>
+                <h1>Profile Page</h1>
```

Comment:
Do we want to label the page as 'Profile Page' I would have thought the landlord's name is a more appropriate title.

if you look at say a youtube video, it wouldn't say 'video page' it just has the title of the video

This tilte is also on the property page not the landlord profile. Is this a mistake?